### ISSUE TITLE
Failed to activate the w3c-validation package

### ISSUE THREAD

**Issue by iambriansreed** [leny/atom-w3c-validation#14]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.0.19
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [w3c-validation](https://github.com/leny/atom-w3c-validation) package, v0.2.0
### Stack Trace

Failed to activate the w3c-validation package

```
At Handler must be a function

Error: Handler must be a function
  at Emitter.module.exports.Emitter.on (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\event-kit\lib\emitter.js:37:15)
  at TextBuffer.module.exports.TextBuffer.onDidChange (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:207:27)
  at C:\Users\breed\.atom\packages\w3c-validation\lib\main.coffee:54:37
  at Workspace.module.exports.Workspace.observeTextEditors (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\workspace.js:276:9)
  at C:\Users\breed\.atom\packages\w3c-validation\lib\main.coffee:53:28
  at Config.module.exports.Config.observeKeyPath (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\config.js:568:7)
  at Config.module.exports.Config.observe (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\config.js:135:21)
  at Object.module.exports.HtmlValidation.activate (C:\Users\breed\.atom\packages\w3c-validation\lib\main.coffee:52:21)
  at Package.module.exports.Package.activateNow (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package.js:245:19)
  at C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package.js:226:30
  at Package.module.exports.Package.measure (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package.js:169:15)
  at Package.module.exports.Package.activate (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package.js:218:14)
  at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package-manager.js:486:21)
  at C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package-manager.js:469:29
  at Config.module.exports.Config.transact (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\config.js:324:16)
  at PackageManager.module.exports.PackageManager.activatePackages (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package-manager.js:464:19)
  at PackageManager.module.exports.PackageManager.activate (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package-manager.js:445:46)
  at Atom.module.exports.Atom.startEditorWindow (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\atom.js:670:21)
  at Object.<anonymous> (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\window-bootstrap.js:12:8)
  at Object.<anonymous> (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\window-bootstrap.js:23:4)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .js] (C:\Users\breed\AppData\Local\atom\app-1.0.19\resources\app.asar\src\compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at setupWindow (file:///C:/Users/breed/AppData/Local/atom/app-1.0.19/resources/app.asar/static/index.js:82:25)
  at window.onload (file:///C:/Users/breed/AppData/Local/atom/app-1.0.19/resources/app.asar/static/index.js:35:9)

```
### Commands

```
```
### Config

``` json
{
  "core": {
    "autoHideMenuBar": true
  },
  "w3c-validation": {
    "validateOnChange": true,
    "hideOnNoErrors": true,
    "useFoldModeAsDefault": true
  }
}
```
### Installed Packages

``` coffee
# User
angularjs, v0.3.0
atom-minify, v0.4.2
language-html, v0.41.3
linter, v1.8.1
linter-htmlhint, v0.1.1
linter-jshint, v1.2.1
w3c-validation, v0.2.0

# Dev
No dev packages
```

**Comment by leny** [comment]
Hi, thanks for the report.
For now, I'm quite busy with some pro work.
I will not be able to review this until january. Feel free to investigate and submit a Pull Request if you find something!

**Comment by leny** [comment]
Hi, sorry for the _long_ time on this one, was very busy with my work.

I've just released a new version (`0.3.0`) of the package, using new modules for validation.

Hope it will fix this issue.
